portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,6 +44,7 @@
   private static final String content = "content";
```

Comment:
I would suggest labeling those params with what they're used for (e.g. DATASTORE_CONTENT_FIELD, or NUM_CONTENTS_PARAM)